projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -40,3 +40,20 @@ export function verifyTicketDetailsPageVisit(){
   cy.url().should('match',/http:\/\/.+\/my\-account\/support\-ticket\/[0-9]+/);
   cy.get('cx-customer-ticketing-messages').should('exist');
 }
+
+export function visitTicketDetailsPageForFirstTicket(){
+  cy.get(`#ticketing-list-table tbody tr:nth-child(${FIRST_TICKET_ROW_INDEX}) .cx-ticketing-list-data:nth-child(${FIRST_TICKET_COLUMN_INDEX}) a.cx-ticketing-list-value`).should('exist');
```

Comment:
can we have the .should in the next line? it seems like a pretty long line

Reply:
I just want the legth of the line to be shorter!

Reply:
I think it can just be 
`cy.get('cx-customer-ticketing-list').get('tbody').get('tr').eq(FIRST_ROW_TICKET_LIST);`

Reply:
Thanks guys. I think the CSS selector works for efficiently. Also, the use of `tbody` here is to differentiate its downstream `tr`s from the ones that may be in `thead`.